apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -277,6 +290,75 @@ describe('[Users]', function () {
 					.end(done);
 			});
 		});
+
+		describe('auto join default channels', () => {
+			let defaultTeamId;
+			let group;
+			let user;
+
+			before(async () => {
+				const teamName = `defaultTeam_${Date.now()}`;
+				user = await createUser({ joinDefaultChannels: true });
+
+				defaultTeamId = await createTeam(credentials, teamName);
+
+				await setDefaultRoom({ roomId: defaultTeamId, roomName: teamName });
+			});
+
+			after(async () => {
+				if (group) {
+					await deleteGroup(group.body.group._id);
+				}
+				await deleteRoom({ roomId: defaultTeamId, type: 'c' });
+				await deleteUser(user);
+			});
+
+			it('should create a subscription for the user in the default channels if the joinDefaultChannels is true', async () => {
+				await request
+					.get(api('subscriptions.getOne'))
+					.set(credentials)
+					.query({ roomId: defaultTeamId })
+					.expect('Content-Type', 'application/json')
+					.expect(200)
+					.expect((res) => {
+						expect(res.body).to.have.property('success', true);
+					});
+			});
+
+			it('should create a subscription for the user in all the auto join channels of the team', async () => {
```

Comment:
Can we create the cases? Do you think it makes sense?

- Create a non-default team and create an auto-join room there (not sure if it's possible though)
- Create a default team with a channel that is not auto-join
- Create a non-default team with a channel that is also not auto-join

Reply:
Done! Thanks for the suggestions